anomalib/models/__init__.py

```diff
@@ -25,7 +25,7 @@
 
 # TODO(AlexanderDokuchaev): Workaround of wrapping by NNCF.
 #                           Can't not wrap `spatial_softmax2d` if use import_module.
-from anomalib.models.padim.model import PadimLightning  # noqa: F401
+from anomalib.models.padim.lightning_model import PadimLightning  # noqa: F401
```

Comment:
Is this import needed?

Reply:
yeah looks like it. But I don't understand how/why it's needed. It is not used anywhere